testing/regress/hpcc/regression/suite.py

```diff
@@ -126,8 +126,17 @@ def addFileToSuite(self, eclfile):
             versions = eclfile.getVersions()
             versionId = 1
             for version in versions:
-                files.append({'basename':basename, 'version':version,  'id':versionId })
-                versionId += 1
+                if 'no'+self.name in version:
+                    # Exclude it from this target
+                    pass
+                else:
+                    # Remove exclusion key(s) from version string
+                    config = getConfig();
```

Comment:
I don't think it is - it looks like two concepts have got confused.

For example I have two hthor clusters on my machine - hthor and hthor2

If I do not have hthor2 in the cluster list I get the following error:

```
gavin:~/dev/hpcc/testing/regress$ ./ecl-test query 'bcd1.ecl' -t=hthor2
[Failure] 1. Unknown target cluster:'hthor2'!
[Error] Error (4000): Unknown cluster! 
^CTraceback (most recent call last):
  File "./ecl-test", line 274, in <module>
    regressMain.main()
  File "./ecl-test", line 269, in main
    self.regress.StopTimeoutThread()
 ```

If I add //nohthor to a test and run 
```./ecl-test query test.ecl -t=hthor```
then it isn't executed.

But if I run 

If I add //nohthor to a test and run 
```./ecl-test query test.ecl -t=hthor```
then it is.

That isn't correct.  The cluster is the name that is supplied to the -t=xxx.  The value used in //nohthor etc. is the platform type (which probably needs to be derived from the environment, or added to the cluster information).